apps/etterlatte-saksbehandling-ui/client/src/components/person/grunnlagshendelser/Grunnlagsendringshendelser.tsx

```diff
@@ -1,94 +1,182 @@
+    return (
+      <>
+        <strong>{props.tittel}</strong>
+        <p>Ingen</p>
+      </>
+    )
+  }
+  return (
+    <>
+      <strong>{props.tittel}</strong>
+      <ul style={{ listStyle: 'none' }}>
+        {props.land.map(({ land, dato }, i) => (
+          <li key={i}>
+            {land} - {formaterKanskjeStringDato(dato)}
+          </li>
+        ))}
+      </ul>
+    </>
+  )
+}
 
-const VisDoedshendelse = ({ doedshendelse }: { doedshendelse: Doedshendelse }) => (
-  <p>
-    {doedshendelse.avdoedFnr} døde {formaterKanskjeStringDato(doedshendelse.doedsdato)}
-  </p>
-)
+const VisUtlandSamsvar = (props: { data: UtlandSamsvar }) => {
+  return (
+    <HendelseSammenligning>
+      <dt>Ut-/innflyttinger i grunnlag</dt>
+      <dd>
+        <LandListeMedDatoOgOverskrift
+          tittel={'Utflyttinger'}
+          land={
+            props.data.fraGrunnlag?.utflyttingFraNorge?.map(({ tilflyttingsland, dato }) => ({
+              land: tilflyttingsland,
+              dato,
+            })) ?? []
+          }
+        />
+        <LandListeMedDatoOgOverskrift
+          tittel={'Innflyttinger'}
+          land={
+            props.data.fraGrunnlag?.innflyttingTilNorge?.map(({ fraflyttingsland, dato }) => ({
+              land: fraflyttingsland,
+              dato,
+            })) ?? []
+          }
+        />
+      </dd>
+      <dt>Ut-/innflyttinger i PDL</dt>
+      <dd>
+        <LandListeMedDatoOgOverskrift
+          tittel={'Utflyttinger'}
+          land={
+            props.data.fraPdl?.utflyttingFraNorge?.map(({ tilflyttingsland, dato }) => ({
+              land: tilflyttingsland,
+              dato,
+            })) ?? []
+          }
+        />
+        <LandListeMedDatoOgOverskrift
+          tittel={'Innflyttinger'}
+          land={
+            props.data.fraPdl?.innflyttingTilNorge?.map(({ fraflyttingsland, dato }) => ({
+              land: fraflyttingsland,
+              dato,
+            })) ?? []
+          }
+        />
+      </dd>
+    </HendelseSammenligning>
+  )
+}
+
+const VisAnsvarligeForeldreSamsvar = (props: { data: AnsvarligeForeldreSamsvar }) => {
+  return (
+    <HendelseSammenligning>
+      <dt>Ansvarlige foreldre i grunnlag</dt>
+      <dd>{props.data.fraGrunnlag?.join(', ') ?? 'Ingen ansvarlige foreldre'}</dd>
+      <dt>Ansvarlige foreldre i PDL</dt>
+      <dd>{props.data.fraPdl?.join(', ') ?? 'Ingen ansvarlige foreldre'}</dd>
+    </HendelseSammenligning>
+  )
+}
 
-const HendelseVisning = (props: { data?: Grunnlagsinformasjon }) => {
+const HendelseVisning = (props: { data?: SamsvarMellomGrunnlagOgPdl }) => {
+  const type = props.data?.type
   switch (props.data?.type) {
-    case 'UTFLYTTING':
-      return <VisUtflytting utflytting={props.data.hendelse} />
-    case 'GJENLEVENDE_FORELDER_DOED':
-    case 'SOEKER_DOED':
-    case 'SOESKEN_DOED':
-      return <VisDoedshendelse doedshendelse={props.data.hendelse} />
+    case 'DOEDSDATO':
```

Comment:
Vi skal ikke ha enums for disse konstantene?

Reply:
De fins som typen GrunnlagsendringsType

Reply:
Sounds good